feature/gribi/otg_tests/supervisor_failure_test/supervisor_failure_test.go

```diff
@@ -302,7 +310,8 @@ func TestSupFailure(t *testing.T) {
 	// Program a route and ensure AFT telemetry returns FIB_PROGRAMMED
 	routeInstall(ctx, t, args)
 	// Verify that static route(203.0.113.0/24) to ATE port-2 is preferred by the traffic.`
-	sendTraffic(t, args.ate)
+	t.Logf("Starting traffic")
+	ate.OTG().StartTraffic(t)
 	time.Sleep(15 * time.Second)
```

Comment:
Add a call to StopTraffic here and remove the stopTraffic function at line 178.

Reply:
Done